src/pages/Calculator.tsx

```diff
@@ -26,6 +26,14 @@ const localStorage = window.localStorage
 const FORM_STATE_KEY = 'formData'
 
 export const Calculator = (): React.ReactElement => {
+  // Mount
+  useEffect(() => window.addEventListener('scroll', scrollListener, true), [])
+  // Unmount
+  useEffect(
```

Comment:
One too many useEffects; just return the unmount handler from the first call

https://reactjs.org/docs/hooks-reference.html

Reply:
Done!